conf/rnaseq_dream.config

```diff
@@ -0,0 +1,35 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running RNA-seq analysis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines settings specific to RNA-seq analysis
+
+    Use as follows:
+        nextflow run nf-core/differentialabundance -profile rnaseq_dream,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+
+    config_profile_name        = 'RNA-seq profile with `VARIANCEPARTITION_DREAM`'
+    config_profile_description = 'Settings for RNA-seq analysis with `VARIANCEPARTITION_DREAM`'
+
+    // Study
+    study_type = 'rnaseq'
+    study_abundance_type = 'counts'
+
+    // Observations
+    observations_id_col        = 'sampleId'
+    observations_name_col      = 'sampleId'
+
+    // Differential options
+    differential_use_dream           = true
```

Comment:
This is OK for now. I think once we get @suzannejin 's work in , this will be simplified like this https://github.com/nf-core/differentialabundance/blob/61dea79f900eb7f0162feb070f5f32eaa833df28/nextflow.config#L98 (see #443)